packages/block-editor/src/components/contrast-checker/index.js

```diff
@@ -34,6 +36,13 @@ function ContrastChecker( {
 	const msg = tinyBackgroundColor.getBrightness() < tinyTextColor.getBrightness() ?
 		__( 'This color combination may be hard for people to read. Try using a darker background color and/or a brighter text color.' ) :
 		__( 'This color combination may be hard for people to read. Try using a brighter background color and/or a darker text color.' );
+	useEffect( () => {
+		if ( tinyBackgroundColor.getBrightness() < tinyTextColor.getBrightness() ) {
+			speak( __( 'This color combination may be hard for people to read. Try using a darker background color and/or a brighter text color.' ) );
+		} else {
+			speak( __( 'This color combination may be hard for people to read. Try using a brighter background color and/or a darker text color.' ) );
+		}
+	}, [ tinyBackgroundColor, tinyTextColor ] );
```

Comment:
If I understand properly both these variables are `tinycolor` objects that are regenerated on each render which means the effect will still be called on each render, we should probably replace those with the underlying props `backgroundColor` and `textColor`

Reply:
You are right, I should use the props instead of the objects (). Updated!